src/renderer.rs

```diff
@@ -138,12 +139,35 @@ pub fn page(
                     a.back href="#top" {
                         (arrow_up())
                     }
+                    @if !hide_version_footer {
+                        (version_footer())
+                    }
                 }
             }
         }
     }
 }
 
+// Get version info
+fn get_version_info() -> String {
+    const PROG_NAME: Option<&'static str> = option_env!("CARGO_PKG_NAME");
```

Comment:
Small nit: You can use `crate_version!()` and `crate_name!()` from clap (which is re-exported from structopt which we already use). This should cut down on the code a little.